sdk/python/examples/bayesianoptimization-katib-sdk.ipynb

```diff
@@ -0,0 +1,543 @@
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'apiVersion': 'kubeflow.org/v1alpha3',\n",
+       " 'kind': 'Experiment',\n",
+       " 'metadata': {'creationTimestamp': '2020-03-04T10:46:49Z',\n",
+       "  'finalizers': ['update-prometheus-metrics'],\n",
+       "  'generation': 1,\n",
+       "  'name': 'bayesianoptimization',\n",
+       "  'namespace': 'anonymous',\n",
+       "  'resourceVersion': '7034918',\n",
+       "  'selfLink': '/apis/kubeflow.org/v1alpha3/namespaces/anonymous/experiments/bayesianoptimization',\n",
+       "  'uid': '7360157b-5e05-11ea-8d71-42010aa00012'},\n",
+       " 'spec': {'algorithm': {'algorithmName': 'bayesianoptimization',\n",
+       "   'algorithmSettings': [{'name': 'random_state', 'value': '10'}]},\n",
+       "  'maxFailedTrialCount': 3,\n",
+       "  'maxTrialCount': 5,\n",
+       "  'metricsCollectorSpec': {'collector': {'kind': 'StdOut'}},\n",
+       "  'objective': {'goal': 0.9999,\n",
+       "   'objectiveMetricName': 'Validation-accuracy',\n",
+       "   'type': 'maximize'},\n",
+       "  'parallelTrialCount': 5,\n",
+       "  'parameters': [{'feasibleSpace': {'list': ['sgd', 'adam', 'ftrl']},\n",
+       "    'name': '--optimizer',\n",
+       "    'parameterType': 'categorical'},\n",
+       "   {'feasibleSpace': {'max': '0.03', 'min': '0.01'},\n",
+       "    'name': '--lr',\n",
+       "    'parameterType': 'double'},\n",
+       "   {'feasibleSpace': {'max': '5', 'min': '2'},\n",
+       "    'name': '--num-layers',\n",
+       "    'parameterType': 'int'}],\n",
+       "  'trialTemplate': {'goTemplate': {'rawTemplate': 'apiVersion: \"batch/v1\"\\nkind: Job\\nmetadata:\\n  name: {{.Trial}}\\n  namespace: {{.NameSpace}}\\nspec:\\n  template:\\n    spec:\\n      containers:\\n      - name: {{.Trial}}\\n        image: docker.io/kubeflowkatib/mxnet-mnist\\n        command:\\n        - \"python3\"\\n        - \"/opt/mxnet-mnist/mnist.py\"\\n        - \"--batch-size=64\"\\n        {{- with .HyperParameters}}\\n        {{- range .}}\\n        - \"{{.Name}}={{.Value}}\"\\n        {{- end}}\\n        {{- end}}\\n      restartPolicy: Never'}}},\n",
+       " 'status': {'completionTime': None,\n",
+       "  'conditions': [{'lastTransitionTime': '2020-03-04T10:46:49Z',\n",
+       "    'lastUpdateTime': '2020-03-04T10:46:49Z',\n",
+       "    'message': 'Experiment is created',\n",
+       "    'reason': 'ExperimentCreated',\n",
+       "    'status': 'True',\n",
+       "    'type': 'Created'}],\n",
+       "  'currentOptimalTrial': {'bestTrialName': '',\n",
+       "   'observation': {'metrics': None},\n",
+       "   'parameterAssignments': None},\n",
+       "  'startTime': '2020-03-04T10:46:49Z'}}"
+      ]
+     },
+     "execution_count": 32,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "kclient.get_experiment(name=\"bayesianoptimization\", namespace=None)"
```

Comment:
I think we can use `namespace=namespace` here, like you did with other API calls to be consistent.

Reply:
updated